source/codegen/metadata/nifgen/functions.py

```diff
@@ -1140,6 +1140,41 @@
         ],
         'returns':'ViStatus'
     },
+    'CreateWaveformComplexF64':{
```

Comment:
Are any of the fields here new compared to base/nimi-python metadata? I guess atleast `grpc_type` is. Document these in CHANGES.md. Same for the function below.

Reply:
Documented the changes.